docs/core/tools/dotnet-nuget-why.md

```diff
@@ -0,0 +1,78 @@
+---
+title: dotnet nuget why command
+description: The 'dotnet nuget why' command provides the user to view the dependency graph of a package.
+ms.date: 08/04/2022
+---
+# dotnet nuget why
+
+**This article applies to:** ✔️ .NET Core 3.1 SDK and later versions
```

Comment:
Which preview is this in? We will want to make sure it doesn't get merged before it's available, and this applies-to text should be updated to .NET 7 Preview SDK.